.travis.yml

```diff
@@ -27,6 +27,8 @@ cache:
     - $HOME/.composer/cache/files
 
 before_install:
+  - if [[ "$TRAVIS_PHP_VERSION" != "hhvm" ]]; then phpenv config-rm xdebug.ini; fi
```

Comment:
Could you add a comment above here explaining that hhvm doesn't support `phpenv`?  Otherwise it's not obvious why the if statement is there.